<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MATES入力ファイルエディタ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="map_editor.qrc">
    <normaloff>:/images/map_editor.png</normaloff>:/images/map_editor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFileOption"/>
    <addaction name="separator"/>
    <addaction name="actionFileTestData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>編集(&amp;E)</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionLarge"/>
    <addaction name="separator"/>
    <addaction name="actionIntsec"/>
    <addaction name="separator"/>
    <addaction name="actionRoad"/>
    <addaction name="actionRoadRepl"/>
    <addaction name="separator"/>
    <addaction name="actionLane2"/>
    <addaction name="actionLane4"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionDirJump"/>
    <addaction name="actionJump"/>
    <addaction name="separator"/>
    <addaction name="actionConfigEditor"/>
   </widget>
   <widget class="QMenu" name="menu_I">
    <property name="title">
     <string>情報(&amp;I)</string>
    </property>
    <addaction name="actionAllInfo"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_I"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionLarge"/>
   <addaction name="separator"/>
   <addaction name="actionIntsec"/>
   <addaction name="separator"/>
   <addaction name="actionRoad"/>
   <addaction name="actionRoadRepl"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionLane2"/>
   <addaction name="actionLane4"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionConfigEditor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileNew">
   <property name="text">
    <string>新規(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新規</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="text">
    <string>開く(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>開く</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>名前をつけて保存(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>名前をつけて保存</string>
   </property>
  </action>
  <action name="actionFileOption">
   <property name="text">
    <string>設定(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>設定</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>終了(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>終了</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>マップエディタについて(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>マップエディタについて</string>
   </property>
  </action>
  <action name="actionIntsec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_intsec.png</normaloff>:/images/tool_intsec.png</iconset>
   </property>
   <property name="text">
    <string>交差点(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>交差点の新規作成／編集／削除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRoad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_road.png</normaloff>:/images/tool_road.png</iconset>
   </property>
   <property name="text">
    <string>単路(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>単路の新規作成／編集／削除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRoadRepl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_road_repl.png</normaloff>:/images/tool_road_repl.png</iconset>
   </property>
   <property name="text">
    <string>単路付け替え(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>単路の付け替え／編集／削除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_zoom_in.png</normaloff>:/images/tool_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>拡大(&amp;Z)</string>
   </property>
   <property name="toolTip">
    <string>画面の拡大</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_zoom_out.png</normaloff>:/images/tool_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>縮小(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>画面の縮小</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionLane2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/line2.png</normaloff>:/images/line2.png</iconset>
   </property>
   <property name="text">
    <string>往復2車線(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>往復2車線(1+1、新規単路)</string>
   </property>
  </action>
  <action name="actionLane4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/line4.png</normaloff>:/images/line4.png</iconset>
   </property>
   <property name="text">
    <string>往復4車線(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>往復4車線(2+2、新規単路)</string>
   </property>
  </action>
  <action name="actionFileTestData">
   <property name="text">
    <string>テストデータ(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>テストデータ</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_select.png</normaloff>:/images/tool_select.png</iconset>
   </property>
   <property name="text">
    <string>選択(S)</string>
   </property>
   <property name="toolTip">
    <string>選択</string>
   </property>
  </action>
  <action name="actionJump">
   <property name="text">
    <string>交差点ジャンプ(&amp;J)</string>
   </property>
   <property name="toolTip">
    <string>交差点ジャンプ</string>
   </property>
  </action>
  <action name="actionConfigEditor">
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_edit.png</normaloff>:/images/tool_edit.png</iconset>
   </property>
   <property name="text">
    <string>設定エディタ起動(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>設定エディタ起動</string>
   </property>
  </action>
  <action name="actionLarge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map_editor.qrc">
     <normaloff>:/images/tool_large.png</normaloff>:/images/tool_large.png</iconset>
   </property>
   <property name="text">
    <string>広域表示(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>広域表示</string>
   </property>
  </action>
  <action name="actionDirJump">
   <property name="text">
    <string>交差点指定ジャンプ(&amp;G)</string>
   </property>
   <property name="toolTip">
    <string>交差点指定ジャンプ</string>
   </property>
  </action>
  <action name="actionAllInfo">
   <property name="text">
    <string>全体情報(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>全体情報</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="map_editor.qrc"/>
 </resources>
 <connections/>
</ui>
